app/src/main/java/com/github/libretube/enums/SupportedClient.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.libretube.enums
+
+enum class ImportFormat(val value: Int) {
+    NEWPIPE(0),
+    FREETUBE(1),
+    YOUTUBECSV(2);
+
+    companion object {
+        fun fromInt(value: Int) = ImportFormat.values().first { it.value == value }
```

Comment:
I think that should be dropped, and instead we should use
```kotlin
ImportFormat.values()[value]
```
which should just work the same way.